internal/cmd/controller/controllers/git/git.go

```diff
@@ -611,13 +627,33 @@ func volumes(
 				},
 			},
 		},
+		{
+			Name: emptyDirTmpVolumeName,
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{},
+			},
+		},
+		{
+			Name: emptyDirHomeVolumeName,
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{},
+			},
+		},
 	}
 
 	volumeMounts := []corev1.VolumeMount{
 		{
-			Name:      "config",
+			Name:      configVolumeName,
 			MountPath: "/run/config",
 		},
+		{
+			Name:      emptyDirTmpVolumeName,
+			MountPath: "/tmp",
+		},
+		{
+			Name:      emptyDirHomeVolumeName,
+			MountPath: "/home/fleet-apply",
```

Comment:
Discussed offline: I believe we should not assume the user running the container will configure `/home/fleet-aply` as the home. Better to explicitly set the`HOME` env. variable as well

Reply:
added in https://github.com/rancher/fleet/pull/1875/commits/73b3e1ae0f97c2a6fcb869009ffa9535739dda87